Airport/Program.cs

```diff
@@ -0,0 +1,577 @@
+﻿
+using System;
+using System.Threading;
+using System.Text.Json;
+using System.Linq;
+using System.Xml;
+
+
+namespace Airport
+{
+    public class Program
+    {
+        readonly static string path = @"Json\";
+        readonly static string jsonPath = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.FullName, path);
+        public static int id = 0;
+        //Deserialize Flight
+        readonly static string filenameFlights = $"{jsonPath}flights.json";
+        readonly static string jsonTextFlights = File.ReadAllText(filenameFlights);
+        readonly static List<Flights> flights = JsonSerializer.Deserialize<List<Flights>>(jsonTextFlights);
+        //Deserialize Passenger
+        readonly static string filePassengers = $"{jsonPath}passenger.json";
+        readonly static string jsonTextPass = File.ReadAllText(filePassengers);
+        readonly static List<Passengers> passengers = JsonSerializer.Deserialize<List<Passengers>>(jsonTextPass);
+        //Deserialize AirCarrier
+        readonly static string filename1 = $"{jsonPath}airCarriers.json";
+        readonly static string jsonTextAir = File.ReadAllText(filename1);
+        readonly static List<AirCarrier> airCarriers = JsonSerializer.Deserialize<List<AirCarrier>>(jsonTextAir);
+
+
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                InitialMenu();
+            }
+            catch (IndexOutOfRangeException ex)
+            {
+                ExceptionMethod(ex);
+            }
+            catch (FormatException ex)
+            {
+                ExceptionMethod(ex);
+            }
+        }
+
+        public static void ExceptionMethod(Exception e)
+        {
+            Console.WriteLine(e.Message);
+            Console.WriteLine(e.StackTrace);
+        }
+
+        public static void InitialMenu()
```

Comment:
In such case better to create MenuHelper for menu, in Program.sc nust be only Main method